worker/echo/systemd/com.redhat.Yggdrasil1.Worker1.echo.service.in

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=yggdrasil echo worker service
+Documentation=https://github.com/RedHatInsights/yggdrasil
+
+[Service]
+Type=dbus
+User=@user@
+Group=@group@
```

Comment:
... I can see that you use same service user and group for running echo worker service. Shouldn't we have special user & group for echo worker? Something similar I have my in my origin PR: https://github.com/RedHatInsights/yggdrasil/pull/157/files#diff-7eb531046c44fae543db356a1c18ce0ad5c9217049353d04fe87eac8da7f80a4

Reply:
What advantage does it bring to run workers as separate users & groups?

Reply:
It is simple. When worker runs as a same user as yggd, then it has the same access permissions. Example: do we want to allow to any worker to own `com.redhat.Yggdrasil1` D-Bus interface and block starting of yggdrasil service? I guess we do not want.

Of course, it is possible to run this example worker as the same user/group as we use for yggdrasil service and simplify few bits, but I think it is not good to show bad practice in any example. Another example: The MySQL server does not use the same user/group as Apache server use, despite database server "communicates" with HTTP server.